app/router.ts

```diff
@@ -5,10 +5,25 @@ import config from 'ember-get-config';
 
 const Router = EmberRouter.extend({
     metrics: service('metrics'),
+    currentUser: service('currentUser'),
+    features: service('features'),
 
     location: config.locationType,
     rootURL: config.rootURL,
 
+    willTransition(oldInfo, newInfo, transition) {
+        const to = transition.targetName;
```

Comment:
Can we call this `targetRouteName` (or just use `transition.targetName` directly) to make it clear that this is a route name?